src/test/scala/ExampleWorkflowSpec.scala

```diff
@@ -49,15 +50,19 @@ class ExampleWorkflowSpec extends AnyWordSpec with Matchers with BeforeAndAfterA
       val params =
         Map(
           "OUTPUT_FILEPATH" -> outputDirectory,
-          "OUTPUT_FILENAME" -> resultFilename,
-          "SHACL_DIRECTORY" -> shaclDirectory,
-          "SHACL_FILENAME"  -> shaclFilename)
+          "OUTPUT_FILENAME" -> s"$resultFilenamePrefix$resultFilenameSuffix",
+          "SHACL_DIRECTORY" -> shaclDirectoryPath,
+          "SHACL_FILENAME"  -> shaclFilename
+        )
       val is = this.getClass.getClassLoader.getResourceAsStream(exampleWorkflow)
-      val _ = WorkflowManager.runTransformation(is, Some(params), Some(plugins))
+      val _ = WorkflowManager.runTransformation(is, "src/test/resources", Some(params), Some(plugins))
```

Comment:
Is this relative path safe to use? i.e. tests run in both IntelliJ and from the Command Line?

Reply:
Would it be possible instead to retrieve the resource via the classpath?

Reply:
Updated to get the working directory from the classpath.